v-next/hardhat-network-helpers/src/types.ts

```diff
@@ -0,0 +1,34 @@
+interface EthersBigNumberLike {
+  toHexString(): string;
+}
+
+interface BNLike {
+  toNumber(): number;
+  toString(base?: number): string;
+}
+
+export type NumberLike =
+  | number
+  | bigint
+  | string
+  | EthersBigNumberLike
+  | BNLike;
```

Comment:
These are the ones that I think we can remove.

Reply:
The 'ethers' and 'ethereumjs-util' dependencies have been removed in [this commit](https://github.com/NomicFoundation/hardhat/pull/5753/commits/5375f74d50984adfd393b04779f1a9550e6348e1).

Now NumberLike only supports number | bigint | string. Tests have been modified as consequence

Reply:
great